src/test/java/marquez/api/resources/NamespaceResourceTest.java

```diff
@@ -41,11 +41,21 @@
 import marquez.service.models.Namespace;
 import org.junit.Before;
 import org.junit.ClassRule;
+import org.junit.Rule;
 import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.mockito.quality.Strictness;
 
 public class NamespaceResourceTest extends NamespaceBaseTest {
 
-  NamespaceName namespaceName = NamespaceName.fromString(NAMESPACE_NAME);
+  @Rule public MockitoRule rule = MockitoJUnit.rule().strictness(Strictness.STRICT_STUBS);
+
+  @Mock private NamespaceService namespaceService;
```

Comment:
We can remove the usage of `NAMESPACE_SERVICE` and replace all references with the instance created with `@Mock`

Reply:
Nvd, you commented on this case in your PR description 👍